android-components/components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt

```diff
@@ -2544,6 +2544,52 @@ class GeckoEngineSessionTest {
         assertFalse(onExceptionCalled)
     }
 
+    @Test
+    fun reanalyzeProduct() {
```

Comment:
do we typically not follow the "GIVEN WHEN THEN" test name pattern for tests in AC?

Reply:
Not consistently, but I updated it for these tests